tools/run_tests/run_interop_tests.py

```diff
@@ -1315,7 +1317,7 @@ def aggregate_http2_results(stdout):
             for language in languages:
                 for test_case in _TEST_CASES:
                     if not test_case in language.unimplemented_test_cases():
-                        if not test_case in _SKIP_ADVANCED + _SKIP_COMPRESSION:
+                        if not test_case in _SKIP_ADVANCED + _SKIP_COMPRESSION + _SKIP_SPECIAL_STATUS_MESSAGE:
```

Comment:
Seems like this list of three is repeated a whole lot. Any way to factor it out?

Reply:
Options are:
1. Combine them into one or two constants.
2. Create constants for each use cases.

I leanly to trade verbosity to readability and remain current solution.